app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -193,12 +193,12 @@ def test_create_new_experiment_remote_settings_reject(selenium, base_url):
     modal.decline_changes()
 
     # Load home page and wait for experiment to show in the Drafts tab
-    selenium.get(base_url)
+    drafts_tab_url = f"{base_url}?tab=drafts"
+    selenium.get(drafts_tab_url)
     experiment_found = False
     for attempt in range(45):
         try:
-            home = HomePage(selenium, base_url).wait_for_page_to_load()
-            home.tabs[-1].click()
+            home = HomePage(selenium, drafts_tab_url)
```

Comment:
his gets the failing integration test to pass, but it's kind of a weird fix.

The thing that it was tripping on was HomePage(selenium, base_url).wait_for_page_to_load() - and as far as I can tell, that just looks for the presence of .directory-table in the DOM.

As far as I can tell, that element is in the DOM when it fails. And this same method is used in other tests and works. But, for some reason, it consistently fails in this test.

Reply:
/Paging Dr @jrbenny35